src/styles/_catalog.scss

```diff
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: 300ms;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  &__card:hover {
```

Comment:
You can write this selector like &:hover. But, don't forgett about nesting.

Reply:
I can't do it, cause then it would trigger animation not on card but on the whole catalog.

Reply:
Try to nest it in the " &__card " selector. It should work correctly

Reply:
Thx, it helped)